app/helpers/self_service/breadcrumbs_helper.rb

```diff
@@ -62,5 +66,9 @@ def crumb_firm_adviser_new
     def crumb_trading_name_new
       { name: t('self_service.trading_name_new.breadcrumb', firm_name: @firm.registered_name) }
     end
+
+    def crumb_principal_edit
+      { name: t('self_service.principal_edit.breadcrumb', firm_name: @firm.registered_name) }
+    end
   end
 end
```

Comment:
Yes it's true, there are no tests for this helper. All of this was already "going on" when these configuration values were being declared in the view files. Honestly I'm not sure what to do here; there doesn't seem to be a lot of value in testing the individual `crumb_*` methods except from the one with logic in it. We could test the chaining behaviour of the `breadcrumbs_*` methods, but in the end this is also configuration but declared in the form of a bunch of methods. I'm dubious about testing configuration data for the same reason that we don't test that locale keys resolve to expected translation strings. Suggestions welcome.